BUILD.gn

```diff
@@ -712,6 +714,11 @@ action("create_dist") {
       ":electron_locale_dist_resources",
     ]
   }
+  if (is_win) {
+    deps += [
+      "app/win:dist_widevine_resources"
```

Comment:
not a big deal, but you should be able to make this a public dep of dist_resources